src/chain/sanitizers.ts

```diff
@@ -10,6 +10,7 @@ export interface Sanitizers<Return> {
   ltrim(chars?: string): Return;
   normalizeEmail(options?: Options.NormalizeEmailOptions): Return;
   rtrim(chars?: string): Return;
+  replace(values_to_replace: any[], new_value: any): Return;
```

Comment:
Small nitpick:

```suggestion
  replace(values_to_replace: any, new_value: any): Return;
```

And then you can cast the value to an array if not already one, in order to optimise for one value replacements?

Reply:
Sure!